### PR TITLE
feat(storage): add set_metadata sample

### PR DESCRIPTION
**Pull request by AlisskaPie**
Related to #1707

### CODE REVIEW

**Comment by AlisskaPie** [comment]
@tritone @tbpg PTAL.

**Review by tbpg — APPROVED** [review]
LGTM. Will let @frankyn approve & merge.

File: storage/objects/set_metadata.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	defer cancel()
+
+	o := client.Bucket(bucket).Object(object)
+	objectAttrsToUpdate := storage.ObjectAttrsToUpdate{
+		Metadata: map[string]string{
+			"keyToAddOrUpdate": "value",
+		},
+	}
+	if _, err := o.Update(ctx, objectAttrsToUpdate); err != nil {
+		return fmt.Errorf("Object(%q).Update: %v", object, err)
```
Comment (tritone) [inline storage/objects/set_metadata.go:48]:
```suggestion
		return fmt.Errorf("ObjectHandle(%q).Update: %v", object, err)
```

**Review by tritone — COMMENTED** [review]
Sorry I missed this before!

**Review by frankyn — APPROVED** [review]

**Pull request merged**